src/views/preview/presentation.jsx

```diff
@@ -238,7 +238,11 @@ const PreviewPresentation = ({
                                             value={projectInfo.instructions}
                                         /> :
                                         <div className="project-description">
-                                            {decorateText(projectInfo.instructions)}
+                                            {decorateText(projectInfo.instructions, {
+                                                usernames: true,
+                                                hashtags: true,
+                                                scratchLinks: false
```

Comment:
I know that we don't link scratch links in project descriptions on 2.0, but was that a design decision, or just because we didn't go back and add it when linking was added to comments? /cc @carljbowman